circle.yml

```diff
@@ -56,6 +56,10 @@ test:
         parallel: true
     - rsync -uav "build/reports/profile" "$CIRCLE_ARTIFACTS/":
         parallel: true
+    - mkdir -p $CIRCLE_ARTIFACTS/heapdumps:
```

Comment:
not sure you want to run the mkdir in parallel with the find, you probably want to make sure the `$CIRCLE_ARTIFACTS/heapdumps` is created before the `find` runs